ubuild.py

```diff
@@ -10,6 +11,16 @@ def main(build):
     build.packages.install(".", develop=True)
     # we install flask to allow testing the example.
     build.packages.install("flask")
+    # the version of cattrs in pypi is very old.
+    # api has also been changed in master branch,
+    # let's install master branch
+    ROOT = os.path.dirname(sys.executable)
+    CATTR_MASTER_BRANCH = "https://github.com/Tinche/cattrs/tarball/master"
```

Comment:
you can install from git directly:

https://github.com/toumorokoshi/notebooks/blob/master/ubuild.py#L5